TravelFunds/2019.md

```diff
@@ -32,3 +32,4 @@ Eemeli | Aro | Node + JS Interactive 2019 | Collab Summit, presenting | | Montre
 Christian | Bromann | Node + JS Interactive 2019 | Attendance / Co-Organizing (NJSI + Collab Summit) | | Montreal (Canada) | December 9 - 15 | 1.133,32 USD | November 9 2019 | https://github.com/nodejs/admin/pull/439 | | | | |
 Waleed | Ashraf | Node + JS Interactive 2019 | Collab Summit, Presenting | | Montreal (Canada) | December 10 - 15 | $1700 | November 06 2019 | https://github.com/nodejs/admin/pull/436 | | | | |
 Manil | Chowdhury | Collab Summit, Montreal 2019 | Collab Summit & Node + JS Interactive 2019 participation | | Montreal QC, Canada) | 2019-12-10 to 2019-12-15 | USD 1768.16 | 2019-11-10 | https://github.com/nodejs/admin/pull/440 | | | | |
+Stephen | Belanger | Node + JS Interactive 2019 + Collab Summit | Speak at Node + JS Interactive 2019 and participate in Collab Summit | | Montreal QC, Canada | 2019-12-10 to 2019-12-15 | CAD 1250 | 2019-11-18 | https://github.com/nodejs/admin/pull/441 | | | | |
```

Comment:
the PR summary says `CAD 1500`, but here it is `CAD 1250` ?

Reply:
Fixed. Originally went with 1250 because I _think_ I can probably cut it that low, but I'm not certain so I wanted to give myself some breathing room as I'm currently unemployed and therefore don't want to buy tickets until I'm sure I have a way to cover the cost.